src/main/java/InputValidator.java

```diff
@@ -0,0 +1,15 @@
+public class InputValidator {
+    private static InputValidator singleInstance = null;
+
+    private InputValidator() { }
+
+    public static InputValidator getInstance() {
+        if(singleInstance == null)
```

Comment:
There should be a space after `if` and curly braces for the code under `if` branch